model/src/main/scala/org/broadinstitute/dsde/rawls/StringValidationUtils.scala

```diff
@@ -32,16 +32,22 @@ trait StringValidationUtils {
     }
   }
 
-  private lazy val billingProjectNameRegex = "[A-z0-9_-]{6,30}".r
-  def validateBillingProjectName(s: String): Future[Unit] = {
-    if(! billingProjectNameRegex.pattern.matcher(s).matches) {
+  private lazy val googleProjectNameRegex = "[A-z0-9_-]{6,30}".r
+  def validateGoogleProjectName(s: String): Future[Unit] = {
+    if(! googleProjectNameRegex.pattern.matcher(s).matches) {
       val msg = s"Invalid name for billing project. Input must be between 6 and 30 characters in length and may only contain alphanumeric characters, underscores, and dashes."
       Future.failed(new RawlsExceptionWithErrorReport(errorReport = ErrorReport(message = msg, statusCode = StatusCodes.BadRequest)))
     } else {
       Future.successful(())
     }
   }
 
+  // Note that the naming requirements for Google projects and  illing projects are
+  // currently identical, but that may change in the future
+  def validateBillingProjectName(s: String): Future[Unit] = {
```

Comment:
I implemented this so I wouldn't need to conflate the terms in `UserService`: https://github.com/broadinstitute/rawls/pull/1447/files#diff-727d553ef55a7aabb5b0d0bca2e6a6fe44422feef4c7ad5ebb9c86bcd92b6572R315